packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -170,8 +170,8 @@ const AccountOrderDetail = () => {
                             spacing={[0, 3]}
                             divider={
                                 <Divider
-                                    visibility={{base: 'hidden', lg: 'visible'}}
-                                    orientation={{lg: 'vertical'}}
+                                    visibility={{base: 'visible'}}
+                                    orientation="vertical"
```

Comment:
![image](https://github.com/user-attachments/assets/f83bf803-e785-4bd4-a3d3-18b1959ff973)
It seems like chakra will pass what `orientation` prop into `aria-orientation`. We don't need to declare prop as object here because we only want to use 'vertical' line. Changed it back to string would fix the a11y